### DISCUSSION TITLE
How to enable camera in PrimeNG v17

### DISCUSSION THREAD

**Discussion by ovazquezv-ahio** [primefaces/community#2550] [PrimeNG]
Hello, 
I have a requirement to take some photos with camera, and i could not find how to do it with primeng, o hope someone can guide me.
I know how to enable camera in a web application, but i think should exists a better way to do it, in order to avoid make a frankenstain application.
Regards

**Comment by ibrahimkelly** [comment]
> Hello, I have a requirement to take some photos with camera, and i could not find how to do it with primeng, o hope someone can guide me. I know how to enable camera in a web application, but i think should exists a better way to do it, in order to avoid make a frankenstain application. Regards

The camera feature is not yet available in PrimeNG. However, you can utilize pure JavaScript to implement this functionality, along with some necessary dependencies..

**Reply by ibrahimkelly** [reply]
Here is an example of dependency [NGX WebCam](https://github.com/basst314/ngx-webcam)
